### ISSUE TITLE
Change of Listview UI / Preventg misclicks

### ISSUE THREAD

**Issue by serossi** [futo-org/grayjay-android#1362]
### Use Cases

Currently all list entreis are similar to youtube in terms of clickable items.
So we can click the thumbnail and title to load that video and click on creator to go too the creator site 
plus the 3 additional buttons on some list (options, watch later, add to queue)

This works well on a Computer but on the phone it is prone to misclicks. starting video by just scrolling, and loading creator page instead of starting the video is a more than common occurence.

### Proposal

i would propose to reduce the clickable items in list view.

i think it would be better to just click on thumbnail to load video.
click on titles do nothing
click on creator do nothing (to get to the creator site you then need to load a video, alternative this could be put into options too)
click on 3 menu buttons as is

i think this would make the app a lot more robust and cleaner to use

**Comment by MKBontwikkeling** [comment]
Yes I do find it a bit finicky too. I like the functionality though. So as far as I'm concerned it's just UI thing.

**Comment by Zvonimir-FUTO** [comment]
Related to #804 and #1391.

**Comment by serossi** [comment]
> Related to [#804](https://github.com/futo-org/grayjay-android/issues/804) and [#1391](https://github.com/futo-org/grayjay-android/issues/1391).

also #1245
